packages/plus/src/components/ContentCard/index.tsx

```diff
@@ -111,6 +111,11 @@ type ContentCardProps = {
   icon?: ReactNode
   subtitle?: string
   title: string
+  /*
+   * Define the HTML Section Heading element level (h1 - h6) to use for the header
+   */
+  headingTag?: ElementType
```

Comment:
I think we can be more restrictive on that enforcing only heading elements (h1, h2, h3, h4, h5, h6)